client/components/editor.vue

```diff
@@ -447,10 +447,20 @@
           });
 
           if (newContents.length) {
-            // Notify to other components that there is new content.
-            this.$emit('content-changed');
-            // Observe new confirmed steps.
-            if (collab.getVersion(this.$editorView.state) > 0) {
+            // Check if collab was initialized with the first steps.
+            const isCollabInitialized = collab.getVersion(this.$editorView.state) > 0;
```

Comment:
I do not think this is correct. Because it could be that not all initial steps come at the same time from the server, so this code could be called multiple times during initialization. You would suppress here initialization only the first time.

Reply:
I do not think we should worry about those events being dispatched unnecessary.